### ISSUE TITLE
[Bug]: Commands not available after installing plugin

### ISSUE THREAD

**Issue by Rahmerh** [akinsho/flutter-tools.nvim#181]
**Describe the bug**
My issue is simple. When I install this plugin like this: 

```
use({
	"akinsho/flutter-tools.nvim",
	config = function()
		require("flutter-tools").setup({
			flutter_path = "/Users/bas/bin/flutter/bin/flutter",
		})
	end,
})
```

The commands mentioned in the README simply don't get available. I don't get any error logs or anything like it.

**To Reproduce**

See packer use statement above. I added it, ran `PackerSync` and restarted neovim. I don't have any flutter commands available

**Expected behavior**
I expect the commands mentioned in the README to be available when I follow the steps to install.

**System information**

- OS: OSX Monterey 12.5.1
- Flutter version: 3.3.0
- Is flutter in $PATH: yes
- neovim version:  0.8

**Comment by Rahmerh** [comment]
Wow okay.. Would have been nice to know :P

**Comment by akinsho** [comment]
Please feel free to make a PR to add this to the documentation, it's assumed that anyone using this plugin would be doing so *inside a dart file* so explicitly mentioning that it requires dart files seemed unnecessary.

**Comment by Rahmerh** [comment]
Fair enough, I guess I just didn't have that thought, I'll leave it up to you to make this explicit or not.
